src/angular/src/app/spark-core-angular/directives/inputs/formatters/sprk-formatter-ssn/sprk-formatter-ssn.directive.spec.ts

```diff
@@ -0,0 +1,65 @@
+import { Component, DebugElement } from '@angular/core';
+import { TestBed, ComponentFixture } from '@angular/core/testing';
+import { SprkFormatterSsnDirective } from './sprk-formatter-ssn.directive';
+import { By } from '@angular/platform-browser';
+
+@Component({
+  selector: 'test',
+  template: `
+  <input 
+  value="hi"
+  type="text"
+  placeholder="Enter some input."
+  sprk-formatter-ssn>
+  `
+})
+class TestComponent {};
+
+describe('SprkFormatterSsnDirective', () => {
```

Comment:
Should SprkFormatterSsnDirective be SprkFormatterSSNDirective? I'm not sure, but wanted to ask what you thought.